server/config/config.exs

```diff
@@ -58,6 +58,8 @@ db_ip_version =
   %{"ipv4" => :inet, "ipv6" => :inet6}
   |> Map.fetch(System.get_env("DB_IP_VERSION", "") |> String.downcase())
 
+webhook_headers = System.get_env("WEBHOOK_HEADERS")
```

Comment:
Can we start using a config file (yaml or toml) to supply our config options instead of env vars?
That'll also avoid things like this blowing up if a header has a colon (unusual, but not unheard of)